packages/gatsby-source-npm-package-search/src/gatsby-node.js

```diff
@@ -28,8 +48,29 @@ exports.sourceNodes = async (
     hitsPerPage: 1000,
   })
 
-  hits.forEach(hit => {
+  await hits.map(async hit => {
     const parentId = createNodeId(`plugin ${hit.objectID}`)
+
+    if (!hit.readme && hit.repository && hit.repository.host === `github.com`) {
+      try {
+        hit.readme = (await got.get(
+          generateGithubReadmeUrl(hit.repository)
+        )).body
+      } catch (err) {
+        // carry-on
+      }
+    }
+
+    if (!hit.readme) {
+      try {
+        hit.readme = (await got.get(
+          url.resolve(`https://unpkg.com/`, `/${hit.objectID}/README.md`)
```

Comment:
What's the reason for using an unpkg url instead of GitHub?

Reply:
I guess the idea here is unpkg is the fallback to the fallback (GitHub) 😛

Reply:
Maybe I should skip github and go straight from unpkg. I copied the github code from yarn and noticed that we didn't get all Readme files as not all package.json have correct info for monorepos. 

Unpkg is always right so I'll probably drop github